lib/osf-components/addon/components/preprints/taxonomy-top-list/component.ts

```diff
@@ -0,0 +1,46 @@
+import Component from '@glimmer/component';
+import { computed } from '@ember/object';
+import SubjectModel from 'ember-osf-web/models/subject';
+
+interface InputArgs {
+    list: SubjectModel[];
+}
+
+interface PairModel {
+    path: string[];
+    text: string;
+}
+
+export default class TaxonomyTopList extends Component<InputArgs> {
+    routerPrefix = 'http://localhost:4200';
+
+
+    @computed('args.list')
```

Comment:
I don't think you need this in a glimmer component. It's an arg, so it wouldn't be looking for recalculation, otherwise you'd make it a tracked variable. So a straight `get` here without the decorator should work.